onboard/catkin_ws/src/cv/scripts/depthai_mock_camera_feed.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+
+import depthai as dai
+import numpy as np
+import cv2
+import os
+import rospy
+import depthai_camera_connect
+
+path = os.path.dirname(__file__)
+NN_PATH = os.path.join(path, '../assets/bloblol.blob')
+IMAGE_RELATIVE_PATH = os.path.join(path, '../assets/left384.jpg')
+
+
+class DepthAIMockImageStream:
+    """
+    THIS FILE IS INCOMPLETE. It can be used for local testing outside of the docker container since
+    it does not require ROS.
```

Comment:
No, resolving it right now.

Reply:
This is currently under development in the `cv-refactor-mock-camera-feed` branch: https://github.com/DukeRobotics/robosub-ros/tree/cv-refactor-mock-camera-feed, so this will be removed once that is merged